<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window</class>
 <widget class="QMainWindow" name="window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Voice Assistant</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #B0BEC5;</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="text_box">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>600</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #ffffff;
border-radius: 10px;
padding: 7px 6px;</string>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Ожидание команд...</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="recognize_button">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#recognize_button:disabled {
	background-color: #cccccc;
	border-radius: 10px;
	color: #000000;
}

QPushButton#recognize_button {
	background-color: #0077c2;
	border-radius: 10px;
	color: #eeeeee;
}

QPushButton#recognize_button:pressed {
	background-color: #81C784;
}</string>
      </property>
      <property name="text">
       <string>Recognize</string>
      </property>
      <property name="shortcut">
       <string>Return</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
